tools/device_management.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, *args, **kwargs):
             "Create or transform a manifest. "
             "Use {} [command] -h for help on each command."
         ).format(self.parser.prog)
+        subparserActionLength = len(self.parser._subparsers._actions)
+        initParser = self.parser._subparsers._actions[subparserActionLength-1].choices['init']
```

Comment:
Oh we also talked offline about trying to filter based on the type of the subparser action instead of assuming it was at the end of the array, did you have any luck with that?

Reply:
@aashishc1988 ^^^

Reply:
I saw this but unfortunately I wouldnt get around this until later this week as we have some more pressing issues for 5.12 release. Lets keep this PR open, and I will fix this as soon as I can.